opus/application/static_media/js/search.js

```diff
@@ -21,7 +21,7 @@ var o_search = {
         minScrollbarLength: opus.minimumPSLength
     }),
     widgetScrollbar: new PerfectScrollbar("#widget-container" , {
-        suppressScrollX: true,
+        // suppressScrollX: true,
```

Comment:
do we still need this commented out code?

Reply:
I put it back for now, but it shouldn't affect anything because we only allow sortable widgets to move vertically (disable horizontal movement). I was trying get the x-scrolling working for widget area only, but couldn't get it to work on Friday. When screen is narrow and we allow the whole body x-scrolling, it will look pretty ugly (plus we need another ps instance for html tag), so I was trying get the x-scrolling only for widget area in this case. I will come back to work on that later after infiniteScroll.